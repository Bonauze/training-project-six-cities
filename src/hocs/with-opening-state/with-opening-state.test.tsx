import React from 'react';
import renderer from 'react-test-renderer';

import withOpeningState from './with-opening-state';

const MockComponent = () => {
  return <div />;
};

const MockComponentWrapped = withOpeningState(MockComponent);

it(`withOpeningState renders correctly`, () => {
  const tree = renderer.create(
      <MockComponentWrapped />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
