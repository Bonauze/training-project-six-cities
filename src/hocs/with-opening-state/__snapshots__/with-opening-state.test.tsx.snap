// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withOpeningState renders correctly 1`] = `<div />`;
